.readthedocs.yaml

```diff
@@ -16,7 +17,14 @@ build:
       - playwright install chromium
       - python ./docs/scripts/generate_gallery_images.py
 
+sphinx:
+  configuration: docs/conf.py
+  # builder: "dirhtml"
+  fail_on_warning: true
```

Comment:
this is good practice but will probably not work for us, given the long list of "expected" warnings that our build produces. 🤔 maybe we should revisit how we handle that, e.g., using a `nitpick_ignore` in `conf.py` instead of writing warnings to file and using a post-build script to check them against our "expected warnings" lists. Let's see what happens.